internal/dev_server/sdk/store_facade.go

```diff
@@ -0,0 +1,41 @@
+package sdk
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/launchdarkly/ldcli/internal/dev_server/model"
+	"github.com/pkg/errors"
+)
+
+// WriteError writes out a given error if it's known or panics if it isn't.
+// Two assumptions it's making
+//   - a panic handling middleware is in use
+//   - This is in the context of flag delivery which has pretty consistent semantics for what's an error across handlers.
+func WriteError(ctx context.Context, w http.ResponseWriter, err error) {
+	switch {
+	case errors.Is(err, model.ErrNotFound):
+		message := fmt.Sprintf("project, %s, not found", GetProjectKeyFromContext(ctx))
+		log.Println(message)
+		log.Println("You may need to call ldcli dev-server add-project")
```

Comment:
if we know the project key they're trying to use we could be more helpful with the command they'd need to run?
```suggestion
		projectKey := GetProjectKeyFromContext(ctx)
		message := fmt.Sprintf("project, %s, not found", projectKey)
		log.Println(message)
		log.Printf("To add your project to the dev server, call `ldcli dev-server add-project --project %s --source {SOURCE_ENV_KEY}", projectKey)
```

Reply:
Oh! Great idea!